mava/components/updating/checkpointer.py

```diff
@@ -68,7 +68,10 @@ def on_parameter_server_init(self, server: SystemParameterServer) -> None:
         if (old_trainer_steps != server.store.parameters["trainer_steps"]) and (
             self.config.restore_best_net is not None
         ):
-            if server.has(BaseNormalisation):
+            if server.has(BaseNormalisation) and (
+                server.store.global_config.normalise_observations
+                or server.store.global_config.normalise_target_values
+            ):
```

Comment:
Nice catch!

There is a failure case here which is having only 1 of obs norm or value norm components and trying to access both of the config variables, but I think this is such a corner case that it probably doesn't matter.

Reply:
Done!